docs/device_report.md

```diff
@@ -264,16 +264,25 @@ RESULT skip protocol.bacnet.pic Bacnet device not found.
 ```
 --------------------
 Collecting TLS cert from target address %% 10.20.96.164
-IOException unable to connect to server.
+SSLHandshakeException: Unable to complete handshake
 
 --------------------
-security.tls.v3
+security.tls.v1.2
```

Comment:
I think this might need to be security.tls.v1_2  -- since the dot character is used to subsection the various test categories.

Reply:
Updated in all locations where this format was referenced

```
--------------------
security.tls.v1_2
--------------------
Verify the device supports TLS 1.2 (as a client)
--------------------
See log above
--------------------
RESULT pass security.tls.v1_2

--------------------
security.tls.v1_3
--------------------
Verify the device supports TLS 1.3 (as a client)
--------------------
See log above
--------------------
RESULT fail security.tls.v1_3
```